### PR TITLE
Cleaned up `ipfs id` mans

### PR DESCRIPTION
**Pull request by RichardLitt**
License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: core/commands/id.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
-const offlineIdErrorMessage = `ID command fails when run without daemon, we are working to fix this.
+const offlineIdErrorMessage = `Id command fails when run without the daemon; we are working to fix this.
```
Comment (whyrusleeping) [inline core/commands/id.go]: huh, this is no longer true. You can run `ipfs id` without a running daemon just fine

Comment (RichardLitt) [inline core/commands/id.go]: Yeah. You don't even use the daemon anymore; it doesn't hit it at all.

Comment (whyrusleeping) [inline core/commands/id.go]: mind removing this in this PR?

Comment (RichardLitt) [inline core/commands/id.go]: Done.

**Comment by whyrusleeping** [comment]
okay, after seeing the test failures and looking into it a little more, i see why we had that message. Its for if we try to `id` another peer on the network while offline

**Comment by whyrusleeping** [comment]
@RichardLitt rewrote that error message in a way that makes more sense to me. Wanna look it over?

File: core/commands/id.go [hunk 2]
```diff
@@ -19,8 +19,9 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
-const offlineIdErrorMessage = `ID command fails when run without daemon, we are working to fix this.
-In the meantime, please run the daemon if you want to use 'ipfs id':
+const offlineIdErrorMessage = `'ipfs id' currently cannot query information on remote
+peers without a running daemon; we are working to fix this.
+In the meantime, please run the daemon if you want to use 'ipfs id' in that way:
```
Comment (RichardLitt) [inline core/commands/id.go]: "In the meantime, if you want to query remote peers using 'ipfs id', please run the daemon:"

**Comment by RichardLitt** [comment]
LGTM. Might clash with https://github.com/ipfs/go-ipfs/pull/2300.

**Comment by whyrusleeping** [comment]
@RichardLitt wanna rebase #2300?

**Comment by RichardLitt** [comment]
@whyrusleeping Done!

**Pull request merged**